week3/src/component/Header/style.js

```diff
@@ -0,0 +1,19 @@
+import styled from "styled-components";
+
+export const St = {
+  HeaderContainer: styled.header`
+    background-color: red;
+
+    height: 3rem;
+
+    display: flex;
+    justify-content: center;
+    align-items: center;
+  `,
+
+  HeaderText: styled.h1`
+    color: white;
+
+    font-size: 2rem;
+  `,
+};
```

Comment:
이거 이런 식으로 써보는 거 어떤가 해요!
아래처럼 쓰면, indent 관련 린트 충돌도 안 일어나고 자동완성도 알맞게 되더라구요!

```suggestion
const HeaderContainer= styled.header`
    background-color: red;

    height: 3rem;

    display: flex;
    justify-content: center;
    align-items: center;
  `,
const  HeaderText= styled.h1`
    color: white;

    font-size: 2rem;
  `,

const St = {
  HeaderContainer,
  HeaderText
}
export default St;
```

Reply:
늦었지만 감사감사해용🌟🌟 이번과제에 반영할ㄲ요!!!!!